core/domain/collection_services_test.py

```diff
@@ -913,28 +916,74 @@ def test_update_collection_tags(self):
                     'new_value': ['duplicate', 'duplicate']
                 }], 'Add a new tag')
 
+    def test_add_collection_skills(self):
+        # Verify skills table is initially empty
```

Comment:
Nit (consistency): this comment should end with a period to be consistent with other comments in this method.